client/core/bookie.go

```diff
@@ -385,7 +385,7 @@ func (dc *dexConnection) bookie(marketID string) *bookie {
 // syncBook subscribes to the order book and returns the book and a BookFeed to
 // receive order book updates. The BookFeed must be Close()d when it is no
 // longer in use. Use stopBook to unsubscribed and clean up the feed.
-func (dc *dexConnection) syncBook(base, quote uint32) (BookFeed, error) {
+func (dc *dexConnection) syncBook(base, quote uint32) (*orderbook.OrderBook, BookFeed, error) {
```

Comment:
This is funny because originally we have the *OrderBook returned, then we removed it and had the orderbook snap as the first send on the feed (`FreshBookAction` I believe). It is certainly most convenient this way.